src/js/routes/about/About.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import AboutContent from '../../../static/about.html';
+
+const About = () => (
+	<div>
+		<div dangerouslySetInnerHTML={{ __html: AboutContent }} />
```

Comment:
What ever changes should be reflected in `src/js/routes/legend/Legend.jsx` and `src/js/routes/api/Api.jsx`